armeria/src/main/java/brave/example/Backend.java

```diff
@@ -0,0 +1,30 @@
+package brave.example;
+
+import brave.http.HttpTracing;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.brave.BraveService;
+import com.linecorp.armeria.server.logging.LoggingService;
+import java.util.Date;
+
+public final class Backend {
+
+  public static void main(String[] args) {
+    final HttpTracing httpTracing = HttpTracingFactory.create("backend");
+
+    final Server server = Server.builder()
+        .http(9000)
+        .service("/health", (ctx, req) -> HttpResponse.of("ok\n"))
```

Comment:
dummy health endpoint unless armeria has a built-in one

Reply:
`HealthCheckService` might work here. It responds a shiny JSON though.